src/main/java/frc/robot/RobotContainer.java

```diff
@@ -56,10 +66,19 @@ public class RobotContainer {
   private final Telemetry logger = new Telemetry(MaxSpeed);
 
   private final RobotStateManager robotStateManager = new RobotStateManager();
+
+  private SendableChooser<Command> autoChooser;
+  private ShuffleboardTab configTab = Shuffleboard.getTab("Config");
+  private Map<String, Command> autonCommands;
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public RobotContainer() {
     // Configure the trigger bindings
     configureBindings();
+    autonCommands.put("Shoot", autonTest());
+    NamedCommands.registerCommands(autonCommands);
```

Comment:
I feel like we will have multiple named commands.  Shoot, alignHeading, intake?

Should these registerCOmmands be at least in their own method to separate them out?  Even if they are in RobotContainer?

Not sure, point of discussion.  ( I see that they just list them in the pathplanner docs )
https://pathplanner.dev/pplib-named-commands.html

Reply:
Moved to a new method